### ISSUE TITLE
Sort tags are not properly recognized

### ISSUE THREAD

**Issue by upsuper** [navidrome/navidrome#2188]
### Description

Many of my songs locally have proper sort tags for title, album, and artist, because they are all Japanese and I need to put pronunciation into it to make them sort in a reasonable way.

However, when I checked the database, and run
```sql
SELECT sort_title FROM media_file WHERE sort_title != '';
```
I see only two are listed, and both of them are FLAC files. The remaining of my files are probably all either M4A or MP3.

### Expected Behaviour

Those sort tags should be stored properly in the database.

### Steps to reproduce

1. Have a media file scanned.
2. Check the database by running the SQL above.

### Platform information

 - Navidrome version: 0.49.2
 - Browser and version: Firefox 109
 - Operating System: Ubuntu 22.10

### Additional information

As an example, `ffprobe` shows the following for one of the file (which doesn't have its sort tags stored by Navidrome):
```
  Metadata:
    major_brand     : mp42
    minor_version   : 1
    compatible_brands: mp42mp41
    creation_time   : 2020-12-23T03:25:35.000000Z
    sort_album_artist: Oohara Yuiko
    copyright-eng   : 
    title           : 旅人の唄
    artist          : 大原ゆい子
    album_artist    : 大原ゆい子
    album           : 旅人の唄
    track           : 1
    disc            : 1
    composer        : 大原ゆい子
    sort_name       : Tabibito no uta
    sort_artist     : Oohara Yuiko
    copyright       : 2021 TOHO CO.,LTD
    date            : 2021
    sort_album      : Tabibito no uta
    sort_composer   : Oohara Yuiko
    genre           : アニメ
```
You can see that there are sort tags inside.

**Comment by KucharczykL** [comment]
Still interested.

**Comment by deluan** [comment]
`sort` tags should be imported normally by Navidrome, unless `taglib` does not support it. Can you please send me one example file to deluan @ navidrome.org? (Please mention the issue number), and I'll take a look.

Thanks

**Comment by deluan** [comment]
Pinging @upsuper: can you provide a sample file that has sort tags and is not imported by Navidrome?

@Cassian-Andor: Please take a look at #2691. It should solve your issue.

**Comment by deluan** [comment]
Hey @upsuper, thanks for the file. Seems that the problem with m4a files was fixed in #2680:

#### Query
```
select artist, sort_artist_name, album, sort_album_name from media_file where sort_artist_name = 'Ohara, Yuiko';
```
#### Result
| artist | sort\_artist\_name | album | sort\_album\_name |
| :--- | :--- | :--- | :--- |
| 大原ゆい子 | Ohara, Yuiko | 旅人の唄 | Tabibito no uta |


This will be in the next release, together with the option to use the sort tags in the UI: #2696
